gpkit/constraints/bounded.py

```diff
@@ -94,10 +95,10 @@ def process_result(self, result):
                 out["value near upper bound"].append(varkey)
         if self.verbosity > 0 and out:
             print
-            print "UNBOUNDED VARIABLES"
+            print "Feasible with these variables bounded:"
```

Comment:
Hmm, I don't buy it, The header states "Feasible with these variables bounded:", and unless I'm missing something, bounding can only correct a dual-infeasible model; bounding cannot make a primal infeasible model feasible.